nomt/src/page_cache.rs

```diff
@@ -1,5 +1,6 @@
 use crate::{
     bitbox::BucketIndex,
+    io::Page as AlignedPage,
```

Comment:
Nit: not super excited about the name since that's a bit of tautology. If you think about it, pages are always aligned, that's kinda whole idea. 

To disambiguate I suggest just using `io::Page`: it's more concise, it's unambigious, it's a bit more greppable.